hedera-node/config.txt

```diff
@@ -23,7 +23,7 @@ app,		HederaNode.jar
 # app, 		PlatformTestingDemo.jar, FCM1KFreeze.json
 # ** END REMOVE FROM SDK RELEASES **
 
- address,  A, Alice,    1, 127.0.0.1, 50204, 127.0.0.1, 50204, 0.0.3
+ address,  0, A, Alice,    1, 127.0.0.1, 50204, 127.0.0.1, 50204, 0.0.3
```

Comment:
hmmm...this may have been from a merge with develop. All the addresses should have node ids now